### ISSUE TITLE
Would we be able to get the .step or .f3d files for this?

### ISSUE THREAD

**Issue by yourbuddydinec** [falkien/Sunlu_S8_Switchwire_Conversion#1]
Hi there. I saw in the `Sunlu_Skirt_Rear_W-Fans_v2.stl` file that the model seemed messed up? Part is floating if printed flat also the middle fan hole seems to start/end abruptly? I would like to be able to change it up and/or fix it if able but the f3d or step file would really help in that. Also wanted to add a knomi mount to the front skirt so it would also help that. Thanks!

Floating part:
![image](https://github.com/user-attachments/assets/f761b95c-42bf-43c1-ba1c-e40d9c1ba34a)
Half hole:
![image](https://github.com/user-attachments/assets/ac6334c0-53e1-4a60-ae85-d716546731e4)
Abrupt start/end
![image](https://github.com/user-attachments/assets/84ba25d5-6b88-4f81-8c5f-55da5fcfd3cb)

**Comment by falkien** [comment]
Yeah when I made those I actually put them together in prusa slicer. They weren't actually made in a 3D CAD program. That's why they're so messed up. I actually don't have step files or CAD files that I could share. I took the STL files and literally put them together in prusa slicer so they would print out. I'm actually going to work on fixing this the proper way sometime soon and we'll gladly share any files that I have or make with you if you're interested.

**Comment by falkien** [comment]
I updated my skirt section. This is a way better setup than I had before and gets rid of all the problems from the previous setup. https://github.com/falkien/Sunlu_S8_Switchwire_Conversion/tree/main/STL's/Skirt_New
